client/src/components/layouts/global.css

```diff
@@ -172,8 +173,8 @@ a:focus {
 .btn-primary:active.focus,
 .btn-primary.active.focus,
 .open > .dropdown-toggle.btn-primary.focus {
```

Comment:
Thanks for catching this. I meant to show this to @ahmadabdolsaheb because the highlighting for some of the buttons on the settings page broke and accidentally committed this file.